### PR TITLE
[data analysis] Add content

### PR DESCRIPTION
**Pull request by emmbow**

### CODE REVIEW

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/bi-tools-looker/how-to-use-looker.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[UX Tips](https://looker.com/ ... 
     ^
  - data-analysis/introduction/bi-tools-looker/intro-to-looker.md: 
 > 	unexpected end of the stream within a single quoted scalar at line 13, column 1:
    
    ^
  - data-analysis/introduction/bi-tools-mode/mode-customizations.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Mode Gallery](https://mode.c ... 
     ^
  - data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md: 
 > 	Cannot compile node with invalid type heading
  - data-analysis/introduction/data-analysis-using-python/python-libraries.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[Install Pandas](https://pand ... 
     ^
  - data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md: 
 > 	Missing compiler for node of type `headline`: `[object Object]`
  - data-analysis/introduction/data-analysis-using-spreadsheets/intro-to-spreadsheets.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Microsoft Excel vs Google Sh ... 
     ^
  - data-analysis/introduction/data-analysis-using-sql/what-is-sql.md: 
 > 	Cannot compile node with invalid type heading
  - data-analysis/introduction/google-analytics/advanced-google-analytics.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Google Analytics Top Tips](h ... 
     ^
  - data-analysis/introduction/google-analytics/key-features.md: 
 > 	No valid answer list in question node
  - data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md: 
 > 	end of the stream or a document separator is expected at line 12, column 3:
    	 - '[The Best Stats You've Ever S ... 
      ^
  - data-analysis/introduction/marketing-analytics/market-analysis.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Total Addressable Market](ht ... 
     ^
  - data-analysis/introduction/product-analytics/a-b-testing.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[How to A/B Test](https://blo ... 
     ^
  - data-analysis/introduction/sales-analytics/boost-your-performance.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[Sales Analytics Dashboards]( ... 
     ^
  - data-analysis/introduction/sales-analytics/other-sales-metrics.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[Sales Metrics](https://www.p ... 
     ^

File: data-analysis/introduction/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - data-analysis-using-spreadsheets
+    - google-analytics
+    - sales-analytics
+    - marketing-analytics
+    - product-analytics
+    - bi-tools-tableau
+    - bi-tools-looker
+    - bi-tools-mode
+    - data-analysis-using-sql
+    - data-analysis-using-python
```
Comment (nem035) [inline data-analysis/introduction/README.md:21]: We should split these into sections, try them out in spider (cc @kapnobatai137)

File: data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+category: must-know
+
+links:
+
+	- '[Filtering Data](https://cmdlinetips.com/2018/02/how-to-subset-pandas-dataframe-based-on-values-of-a-column/){website}'
+
+
+---
+## Content
+
```
Comment (kapnobatai137) [inline data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md:18]: missing section separator and it should be the other way around:
```
---

# Headline

---
## Content
```

File: data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+- Remove multi-line headers
+- Remove titles, formatting, file version, date and authors (use the file history and versioning for that)
+
+2. **Untidy your data**
+
+- Each row must represent one unaggregated measure
+- Each cell must contain only one dimension or measure
+- Each column must have a unique name
+
+# More Tips
```
Comment (kapnobatai137) [inline data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md:40]: - h1 (`#`) is reserved for insight headline
- h2 (`##`) is reserved for section titles (e.g. `## Practice`)

you can use h3 (`###`) inside the content section (`## Content`) meaning this should be
```
### More Tips
```

File: data-analysis/introduction/data-analysis-using-sql/what-is-sql.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+author: emmab
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+---
```
Comment (kapnobatai137) [inline data-analysis/introduction/data-analysis-using-sql/what-is-sql.md:11]: missing insight headline

it should be:

```
---

# Insight Headline

---
## Content
```

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/bi-tools-looker/how-to-use-looker.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[UX Tips](https://looker.com/ ... 
     ^
  - data-analysis/introduction/bi-tools-looker/intro-to-looker.md: 
 > 	unexpected end of the stream within a single quoted scalar at line 13, column 1:
    
    ^
  - data-analysis/introduction/bi-tools-mode/mode-customizations.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Mode Gallery](https://mode.c ... 
     ^
  - data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md: 
 > 	Cannot compile node with invalid type heading
  - data-analysis/introduction/data-analysis-using-python/python-libraries.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[Install Pandas](https://pand ... 
     ^
  - data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md: 
 > 	Missing compiler for node of type `headline`: `[object Object]`
  - data-analysis/introduction/data-analysis-using-spreadsheets/intro-to-spreadsheets.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Microsoft Excel vs Google Sh ... 
     ^
  - data-analysis/introduction/data-analysis-using-sql/what-is-sql.md: 
 > 	Cannot compile node with invalid type heading
  - data-analysis/introduction/google-analytics/advanced-google-analytics.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Google Analytics Top Tips](h ... 
     ^
  - data-analysis/introduction/google-analytics/key-features.md: 
 > 	No valid answer list in question node
  - data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[The Best Stats You've Ever S ... 
     ^
  - data-analysis/introduction/marketing-analytics/market-analysis.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[Total Addressable Market](ht ... 
     ^
  - data-analysis/introduction/product-analytics/a-b-testing.md: 
 > 	end of the stream or a document separator is expected at line 13, column 2:
    	- '[How to A/B Test](https://blo ... 
     ^
  - data-analysis/introduction/sales-analytics/boost-your-performance.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[Sales Analytics Dashboards]( ... 
     ^
  - data-analysis/introduction/sales-analytics/other-sales-metrics.md: 
 > 	end of the stream or a document separator is expected at line 12, column 2:
    	- '[Sales Metrics](https://www.p ... 
     ^

File: data-analysis/introduction/bi-tools-looker/how-to-use-looker.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+links:
+
+	- '[Looker Training](https://training.looker.com/looker-for-data-consumers){website}'
```
Comment (kapnobatai137) [inline data-analysis/introduction/bi-tools-looker/how-to-use-looker.md:13]:
```suggestion
  - '[Looker Training](https://training.looker.com/looker-for-data-consumers){website}'
```

File: data-analysis/introduction/bi-tools-looker/how-to-use-looker.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+	- '[UX Tips](https://looker.com/blog/dashboard-ux-ui){website}'
```
Comment (kapnobatai137) [inline data-analysis/introduction/bi-tools-looker/how-to-use-looker.md:14]:
```suggestion
  - '[UX Tips](https://looker.com/blog/dashboard-ux-ui){website}'
```

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/bi-tools-looker/intro-to-looker.md: 
 > 	unexpected end of the stream within a single quoted scalar at line 12, column 1:
    
    ^
  - data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md: 
 > 	Cannot compile node with invalid type heading
  - data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md: 
 > 	Missing compiler for node of type `headline`: `[object Object]`
  - data-analysis/introduction/data-analysis-using-sql/what-is-sql.md: 
 > 	Cannot compile node with invalid type heading
  - data-analysis/introduction/google-analytics/key-features.md: 
 > 	No valid answer list in question node
  - data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md: 
 > 	bad indentation of a sequence entry at line 12, column 26:
      - '[The Best Stats You've Ever Seen](https://www.youtub ... 
                             ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/bi-tools-looker/intro-to-looker.md: 
 > 	unexpected end of the stream within a single quoted scalar at line 12, column 1:
    
    ^
  - data-analysis/introduction/google-analytics/key-features.md: 
 > 	No valid answer list in question node
  - data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md: 
 > 	bad indentation of a sequence entry at line 12, column 26:
      - '[The Best Stats You've Ever Seen](https://www.youtub ... 
                             ^

File: data-analysis/introduction/bi-tools-looker/intro-to-looker.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+links:
+  - '[Looker vs Tableau](https://www.quora.com/To-anyone-that-has-used-Looker-how-would-you-compare-it-to-Tableau-in-terms-of-price-capabilities){website}
```
Comment (kapnobatai137) [inline data-analysis/introduction/bi-tools-looker/intro-to-looker.md:12]: missing end quote :smile: 
```suggestion
  - '[Looker vs Tableau](https://www.quora.com/To-anyone-that-has-used-Looker-how-would-you-compare-it-to-Tableau-in-terms-of-price-capabilities){website}'
```

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/google-analytics/key-features.md: 
 > 	No valid answer list in question node
  - data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md: 
 > 	bad indentation of a sequence entry at line 12, column 26:
      - '[The Best Stats You've Ever Seen](https://www.youtub ... 
                             ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md: 
 > 	bad indentation of a sequence entry at line 12, column 26:
      - '[The Best Stats You've Ever Seen](https://www.youtub ... 
                             ^

**Comment by enki-hq** [comment]
Rule insight:slug failed for files: 
  - data-analysis/introduction/data-analysis-using-sql/what-is-sql.md

**Comment by enki-hq** [comment]
Rule workout:insights failed for files: 
  - data-analysis/introduction/data-analysis-buzzwords/README.md
  - data-analysis/introduction/data-analysis-terminology/README.md

**Comment by enki-hq** [comment]
Rule topic:data failed for files: 
  - data-analysis/README.md

**Comment by enki-hq** [comment]
Rule topic:data failed for files: 
  - data-analysis/README.md

**Comment by enki-hq** [comment]
Rule topic:data failed for files: 
  - data-analysis/README.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/alerts-and-sharing.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/alerts-and-sharing.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/how-to-use-looker.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/how-to-use-looker.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/intro-to-looker.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/intro-to-looker.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/more-advanced-looker.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-looker/more-advanced-looker.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/how-to-use-mode.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/how-to-use-mode.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/intro-to-mode.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/intro-to-mode.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/mode-customizations.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/mode-customizations.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/mode-for-data-analysts.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-mode/mode-for-data-analysts.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/dashboards-and-stories.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/dashboards-and-stories.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/intro-to-tableau.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/intro-to-tableau.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/make-your-results-stand-out.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/make-your-results-stand-out.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/tableau-desktop-ui.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/bi-tools-tableau/tableau-desktop-ui.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/big-data.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/big-data.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/key-metrics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/key-metrics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/north-star-metric.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/north-star-metric.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/relative-vs-absolute-metrics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-buzzwords/relative-vs-absolute-metrics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-terminology/basic-terminology-i.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-terminology/basic-terminology-i.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-terminology/basic-terminology-ii.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-terminology/basic-terminology-ii.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-terminology/more-terminology.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-terminology/more-terminology.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/intro-to-python-for-data-analysis.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/intro-to-python-for-data-analysis.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/pandas-functions-i.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/pandas-functions-i.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/pandas-functions-ii.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/python-libraries.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/python-libraries.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/using-jupyter-notebooks.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-python/using-jupyter-notebooks.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/conditional-formatting.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/conditional-formatting.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/data-visualization.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/data-visualization.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/intro-to-spreadsheets.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/intro-to-spreadsheets.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/pivot-tables.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/pivot-tables.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/sorting-and-filtering.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-spreadsheets/sorting-and-filtering.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/da-what-is-sql.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/da-what-is-sql.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/sql-tools.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/sql-tools.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/syntax-i.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/syntax-i.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/syntax-ii.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/syntax-ii.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/syntax-iii.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/data-analysis-using-sql/syntax-iii.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/google-analytics/advanced-google-analytics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/google-analytics/advanced-google-analytics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/google-analytics/google-analytics-user-interface.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/google-analytics/google-analytics-user-interface.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/google-analytics/how-to-use-google-analytics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/google-analytics/how-to-use-google-analytics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/google-analytics/key-features.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/google-analytics/key-features.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/google-analytics/why-google-analytics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/google-analytics/why-google-analytics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/data-analysis-for-everyone.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/data-analysis-process.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/data-analysis-process.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/data-analysis-vs-data-science.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/data-analysis-vs-data-science.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/what-is-data-analysis.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/intro-to-data-analysis/what-is-data-analysis.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/marketing-analytics/intro-to-marketing-analytics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/marketing-analytics/intro-to-marketing-analytics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/marketing-analytics/market-analysis.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/marketing-analytics/market-analysis.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/marketing-analytics/marketing-analytics-tools.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/marketing-analytics/marketing-analytics-tools.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/marketing-analytics/marketing-metrics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/marketing-analytics/marketing-metrics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/marketing-analytics/seo-analytics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/marketing-analytics/seo-analytics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/product-analytics/a-b-testing.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/product-analytics/a-b-testing.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/product-analytics/engagement.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/product-analytics/engagement.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/product-analytics/intro-to-product-analytics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/product-analytics/intro-to-product-analytics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/product-analytics/product-analytics-tools.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/product-analytics/product-analytics-tools.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/product-analytics/retention.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/product-analytics/retention.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/sales-analytics/boost-your-performance.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/sales-analytics/boost-your-performance.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/sales-analytics/other-sales-metrics.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/sales-analytics/other-sales-metrics.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/sales-analytics/pipeline-analysis.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/sales-analytics/pipeline-analysis.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/sales-analytics/sales-analysis-tools.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/sales-analytics/sales-analysis-tools.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)
- [ ] [enkidevs/curriculum/data-analysis/introduction/sales-analytics/your-turn.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/data-analysis/introduction/sales-analytics/your-turn.md?ref=3b6264ff0ee4b6c206d41046676a6e3877c57c46)

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/README.md: 
 > 	bad indentation of a mapping entry at line 18, column 5:
        '2':
        ^

File: data-analysis/introduction/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    - intro-to-data-analysis
+    - data-analysis-terminology
+    - data-analysis-buzzwords
+    - data-analysis-using-spreadsheets
+  '1':
+    - google-analytics
+    - sales-analytics
+    - marketing-analytics
+    - product-analytics
+    '2':
```
Comment (kapnobatai137) [inline data-analysis/introduction/README.md:18]: too much indentation :smile: 
```suggestion
  '2':
```

**Comment by enki-hq** [comment]
Failed parsing files: 
  - data-analysis/introduction/README.md: 
 > 	bad indentation of a mapping entry at line 18, column 5:
        '2':
        ^

**Pull request merged**